airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftCopyDestinationAcceptanceTest.java

```diff
@@ -59,9 +60,19 @@ protected JsonNode getFailCheckConfig() {
                                            final String namespace,
                                            final JsonNode streamSchema)
       throws Exception {
-    return retrieveRecordsFromTable(namingResolver.getRawTableName(streamName), namespace)
+    final List<JsonNode> records = retrieveRecordsFromTable(namingResolver.getRawTableName(streamName), namespace);
+    return records
         .stream()
-        .map(j -> Jsons.deserialize(j.get(JavaBaseConstants.COLUMN_NAME_DATA).asText()))
+        .map(j -> {
+          final String text = j.get(JavaBaseConstants.COLUMN_NAME_DATA).asText();
+          final JsonNode deserialized = Jsons.deserialize(text);
+          if (deserialized.isTextual()) {
+            final JsonNode d = Jsons.deserialize(deserialized.asText());
+            return d;
+          } else {
+            return deserialized;
+          }
+        })
```

Comment:
Did we manage to identify a change in redshift destination behavior that would make the string being serialized twice??